ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/router_isis.schema.yml

```diff
@@ -0,0 +1,197 @@
+    type: dict
+    display_name: Router ISIS
+    keys:
+      instance:
+        type: str
+        description: ISIS Instance Name
+      net:
+        type: str
+        description: CLNS Address like "49.0001.0001.0000.0001.00"
+      router_id:
+        type: str
+        display_name: Router ID
+        description: IPv4 Address
+      is_type:
+        type: str
+        display_name: IS Type
+        valid_values:
+          - "level-1"
+          - "level-1-2"
+          - "level-2"
+      log_adjacency_changes:
+        type: bool
+      mpls_ldp_sync_default:
+        type: bool
+        display_name: MPLS LDP Sync Default
+      timers:
+        type: dict
+        keys:
+          local_convergence:
+            type: dict
+            keys:
+              protected_prefixes:
+                type: bool
+              delay:
+                type: int
+                convert_types:
+                  - str
+                default: 10000
+                description: Delay in milliseconds.
+      advertise:
+        type: dict
+        keys:
+          passive_only:
+            type: bool
+      address_family:
+        type: list
+        items:
+          type: str
+          valid_values:
+            - "ipv4"
+            - "ipv6"
+            - "ipv4 unicast"
+            - "ipv6 unicast"
+          description: Address Family
+      isis_af_defaults:
+        type: list
+        display_name: ISIS AF Defaults
+        items:
+          type: str
+          description: |
+            EOS CLI rendered under the address families
+            Example "maximum-paths 64"
+      redistribute_routes:
+        type: list
+        items:
+          type: dict
+          keys:
+            source_protocol:
+              type: str
+              required: true
+              valid_values:
+                - "bgp"
+                - "connected"
+                - "isis"
+                - "ospf"
+                - "ospfv3"
+                - "static"
+            route_map:
+              type: str
+              description: Route-map name
+            include_leaked:
+              type: bool
+            ospf_route_type:
+              type: str
+              display_name: OSPF Route Type
+              valid_values:
+                - "external"
+                - "internal"
+                - "nssa-external"
+              description: ospf_route_type is required with source_protocols 'ospf' and 'ospfv3'
+      address_family_ipv4:
+        type: dict
+        display_name: Address Family IPv4
+        keys:
+          maximum_paths:
+            type: int
+            convert_types:
+              - "str"
+            min: 1
+            max: 128
+          fast_reroute_ti_lfa:
+            type: dict
+            display_name: Fast Reroute TI LFA
+            keys:
+              mode:
+                type: str
+                valid_values:
+                  - "link-protection"
+                  - "node_protection"
```

Comment:
```suggestion
                  - "node-protection"
```

Reply:
resolved in [09dc0d6](https://github.com/aristanetworks/ansible-avd/pull/2181/commits/09dc0d6725f98a0aece1848e1f0b486eeca1cb61)